gapis/api/service.proto

```diff
@@ -289,6 +289,7 @@ message EnumValue {
 message BitfieldValue {
   repeated uint64 set_bits = 1;
   repeated string set_bitnames = 2;
+  bool is_color = 3;
```

Comment:
Instead of `is_color`, we should call this something more generic. Maybe `short`?

Reply:
What other instances would this exception cover?

Reply:
I don't have a specific case in mind, but I could imagine an XYZW, enum, for example.